src/view/widgets/date_time_picker_widget.py

```diff
@@ -38,4 +38,7 @@ def create(self, config: dict[str, Any]) -> pn.widgets.DatetimePicker:
             onlychanged=True,
         )
         self.controller_instance.register("item_choice", date_time_picker_widget)
+
+        date_time_picker_widget.reset_identifier = c.RESET_IDENTIFIER_ITEM_CHOICE
```

Comment:
Konfigurierbar wäre sinnvoll imho, auch bei den anderen widgets!

Reply:
Dieser Reset Identifier ist für den Conroller um zu wissen, für welchen typ das widget bei ihm registriert ist. Bedeutet zum Beispiel: Ein date time picker widget ist Stand jetzt immer als "item_choice" beim controller registriert (siehe Zeile 40) da das immer so ist, ist der reset type auch immer der gleiche nämlich "item_choice". Ich hab es hier noch nicht gemacht weil nicht unbedingt Scope vom Ticket aber wollte eines schreiben, damit wir mal alle Strings wie diesen in die constants auslagern damit es überall einheitlich ist. Konfigurierbar kann man drüber nachdenken, würde mir jetzt aber kein guter Grund einfallen, einen Date Time Picker als zum Beispiel als item_filter zu registrieren (und weiß nicht ob das einfach so klappt)

Reply:
kk